setup.py

```diff
@@ -49,7 +49,7 @@
     "fs>=2.0",
     "marshmallow>2.13.0,<=2.20.1",
     "pyOpenSSL==17.5.0",
-    "reana-commons[kubernetes]>=0.7.5a2,<0.8.0",
+    "reana-commons[kubernetes]>=0.7.5,<0.8.0",
```

Comment:
BTW note CI failures related to Flask/Werkzeug stack:

```
ImportError: cannot import name 'ContextVar' from 'werkzeug.local' 
```

Reply:
I had to remove flask and werkzeug dependencies like it was done [here](https://github.com/reanahub/reana-server/commit/942da457206a6d581b19c340f2006e4d62ee8c2e) to make it work

Reply:
Hmm, this updates all the dependencies so it may be perhaps dangerous? E.g. alembic 1.6 over 1.4... So we'd have to carefully check whether everything is alright... Alternatively, could we only update the flask/werkzeug requirement family and retain the rest?

Reply:
Nice, didn't know it was possible! I did `pip-compile --output-file=requirements.txt -P flask -P werkzeug -P invenio-oauth2server setup.py` to generate the file

Reply:
Hmm, I'm seeing some troubles with the partial update:

```console
$ kubectl logs reana-server-558bf7557c-4249z rest-api
  File "/usr/local/lib/python3.8/site-packages/invenio_oauthclient/ext.py", line 14, in <module>
    from flask_oauthlib.client import OAuth as FlaskOAuth
  File "/usr/local/lib/python3.8/site-packages/flask_oauthlib/client.py", line 18, in <module>
    from werkzeug import url_quote, url_decode, url_encode
ImportError: cannot import name 'url_quote' from 'werkzeug' (/usr/local/lib/python3.8/site-packages/werkzeug/__init__.py)
unable to load app 0 (mountpoint='') (callable not found or import error)

$  kubectl logs reana-server-558bf7557c-4249z scheduler
  File "/usr/local/lib/python3.8/site-packages/flask_oauthlib/client.py", line 18, in <module>
    from werkzeug import url_quote, url_decode, url_encode
ImportError: cannot import name 'url_quote' from 'werkzeug' (/usr/local/lib/python3.8/site-packages/werkzeug/__init__.py)
```

It was working well before, so pehaps we need to upgrade some more (or some less) packages...

I'll let it rerun again just to confirm/infirm, using latest PR set:

```
$ reana-dev git-fetch -c CLIENT -c CLUSTER --exclude-components r-a-krb5,r-a-vomsproxy
$ reana-dev git-status -c CLIENT -c CLUSTER --exclude-components r-a-krb5,r-a-vomsproxy --base maint-0.7 
Excluding component(s) ['reana-auth-krb5', 'reana-auth-vomsproxy']
reana-workflow-engine-yadage @ pr-195 @ ef5b360 release: 0.7.5
reana-workflow-engine-cwl @ pr-189 @ 4c80981 release: 0.7.6
reana-db @ maint-0.7 @ 327a8ad release: 0.7.3
pytest-reana @ maint-0.7 @ 1329b52 release: 0.7.2
reana-workflow-engine-serial @ pr-146 @ 4963ed2 release: 0.7.5
reana-commons @ maint-0.7 @ f2da55b release: 0.7.5
reana-workflow-controller @ pr-387 @ bb52e34 release: 0.7.4
reana-ui @ maint-0.7 @ 75f39be release: 0.7.2
reana-server @ pr-387 @ 24360a2 release: 0.7.6
reana-job-controller @ pr-319 @ e591e02 release: 0.7.5
reana-client @ pr-532 @ 1460209 release: 0.7.5
reana-message-broker @ maint-0.7 @ 9c7ceac release: 0.7.1
```

Reply:
I confirm the troubles... Perhaps the safest would be to stay on the same Invenio package release series (e.g. see below the oauth change from 1.0 to 1.1) and just limit the flask/werkzeug ourselves?

```diff
-invenio-oauth2server==1.0.5  # via reana-server (setup.py)
+invenio-oauth2server==1.1.0  # via reana-server (setup.py)
```

BTW seeing also:

```diff
-werkzeug==0.16.1          # via flask, flask-kvsession-invenio, invenio-oauth2server, reana-commons, reana-server (setup.py)
+werkzeug==2.0.1
```

So that explains the troubles... We need a werkzeug pin to be less than 2...